### PR TITLE
Factor out common binary-init stuff

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: cpp/util/util.cc [hunk 1]
```diff
@@ -191,4 +199,27 @@ string ToBase64(const string& from) {
   return ret;
 }
 
+void InitCT(int* argc, char** argv[]) {
+  google::SetVersionString(cert_trans::kBuildVersion);
+  google::ParseCommandLineFlags(argc, argv, true);
+  google::InitGoogleLogging(*argv[0]);
+  google::InstallFailureSignalHandler();
+
+
```
Comment (pphaneuf) [inline cpp/util/util.cc]: Spurious line.

Comment (AlCutter) [inline cpp/util/util.cc]: Zapped

File: cpp/client/ct.cc [hunk 2]
```diff
@@ -981,10 +981,8 @@ int Monitor() {
 // Exit code upon abnormal exit (CHECK failures): != 0
 // (on UNIX, 134 is expected)
 int main(int argc, char** argv) {
+  util::InitCT(&argc, &argv);
```
Comment (pphaneuf) [inline cpp/client/ct.cc]: Pretty sure you have to set the usage message *before* calling `google::ParseCommandLineFlags`?

File: cpp/util/util.cc [hunk 3]
```diff
@@ -191,4 +199,5 @@ string ToBase64(const string& from) {
   return ret;
 }
 
+
```
Comment (pphaneuf) [inline cpp/util/util.cc]: All that I like two lines between top-level declarations, I think this file can probably be dropped from your PR? :wink:

**Comment by pphaneuf** [comment]
Done with this review.

File: cpp/client/ct.cc (hunk 2 above)
Comment (AlCutter) [inline cpp/client/ct.cc]: Good catch, done.

File: cpp/util/util.cc (hunk 3 above)
Comment (AlCutter) [inline cpp/util/util.cc]: That line makes all the difference, man.

**Comment by pphaneuf** [comment]
LGTM

**Pull request merged**